### ISSUE TITLE
General Localization - README

### ISSUE THREAD

**Issue by auremoser** [mozillascience/studyGroup-GSOC#8]
## Description

Right now, the [Study Group Repo](https://github.com/mozillascience/studyGroup) documentation is in English, with some (awesome but sporadic) translations in [Indonesian](https://github.com/mozillascience/studyGroup/blob/gh-pages/lesson-template-id.md), [Bengali](https://github.com/mozillascience/studyGroup/blob/gh-pages/bn.yml) and [Portugeuse](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-pt.md) submitted by our community members, we'd love to solicit more languages or improvements on these translations if noted/necessary.

## Process

1. Step 1: Start with the main [Study Group Repo](https://github.com/mozillascience/studyGroup)
2. Step 2: Fork the repository, clone it locally and create a README file to start, which translates the [README]() from English to another language
3. Step 3: Save that file as `README-xx.md` where the `xx` is the [two letter code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for your language of translation. If your language is not include as a two letter code in that list, comment here to ask what the best short code might be.

Look at [this example](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-pt.md) of the readme written in Portugeuse to see an illustration of the steps above.

## Expected result

We will have a collection of READMEs for the Study Group project in a variety of languages

## Current result

Currently we have a [Portuguese](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-pt.md) translation.

## Image

[Portuguese Translation of the README](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-pt.md)
![screen shot 2017-03-22 at 3 31 43 pm](https://cloud.githubusercontent.com/assets/1559703/24216470/af5f3f20-0f12-11e7-8312-c7148894b095.png)

----

## References
* [README for Study Group Project](https://github.com/mozillascience/studyGroup/blob/gh-pages/README.md)
* [Portuguese Translation of the README](https://github.com/mozillascience/studyGroup/blob/gh-pages/README-pt.md)
* [Two-letter codes for Languages](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)

**Comment by Shashi456** [comment]
I'd Like to work on this bug

**Comment by auremoser** [comment]
Great @Shashi456 ! What languages would you like to start translating too? Or how would you like to start working on this? Feel free to create another issue that references this one, with a more specific task outline. 

I created this issue as a broad overview of our localization needs, but it likely needs to be broken down into feasible micro tasks (like "translate the readme into french," or "review and update existing translations in portuguese"). We have updated the readme since the Portuguese translation was submitted, and now I do not have enough fluency in portuguese to update the README-pt.md, for example.

Let me know what you'd like to work on!

**Comment by lvyachao** [comment]
I can cover Chinese version later. (README-zh.md)

**Comment by auremoser** [comment]
Great, thank you @lvyachao !
@AnamikaD started some Chinese translations [here](https://github.com/mozillascience/studyGroup/pull/73/commits/5ab1a96de9716f93a4d40e7d64c4fed86c880f18) if you'd like to take a look and work from there. The pull request was never merged because we dropped the chinese files and she added translations in Hindi.

**Comment by lvyachao** [comment]
@auremoser Yes don't pull request yet lol...I find it did looks funny from the view of a native Chinese speaker... I will handle it after I finish proposal writing...

**Comment by auremoser** [comment]
Sounds good, thanks for your help.
